.github/renovate.json

```diff
@@ -55,7 +55,8 @@
         "oss/kubernetes/kube-proxy",
         "oss/binaries/kubernetes/kubernetes-node",
         "oss/binaries/kubernetes/azure-acr-credential-provider",
-        "containernetworking/azure-npm"
+        "containernetworking/azure-npm",
+        "rpm"
```

Comment:
Why don't we have that for Deb?  I don't get why we have this called "RPM" only

Reply:
Should this be Moby-containerD. Etc.

Reply:
Copied something wrong my fork. Thanks for catching this. Removed "rpm".

I have 1 rule to create PR for minor update for "moby-runc", "moby-containerd", "containerd2" without auto-merge. And another rule with auto-merge for patch update.

Reply:
Is auto-merge working properly though? In the aks-gpu repo it still asks for PR approvals when auto-merge is set to true?